app/create/page.tsx

```diff
@@ -0,0 +1,58 @@
+import { CreatePost } from '@/components/post/create-post';
+import { Profile } from '@/components/user/profile';
+import { UserList } from '@/components/user/user-list';
+import { User } from '@/utils/models';
+import {
+  GetUserById,
+  GetUserFollowees,
+  GetUsers,
+} from '../api/actions/user.actions';
+import { auth } from '../api/auth/[...nextauth]/auth';
+
+export default async function Home() {
```

Comment:
What is the purpose of the create page? I am not quite sure if I understand it.

Reply:
The idea was to redirect new users to this page. but i don't think we will get around to implement new user creation from within the app. I will delete it. We can always reactivate it.

Reply:
I have moved the "Voll leer hier"- component to the _personal_ user page where it will be shown if there are zero posts.